### ISSUE TITLE
Cannot find code for p!help command

### ISSUE THREAD

**Issue by oliver408i** [VasilisThePikachu/Private-Channel#1]
Title says it all. I would like to modify it so I can write the arguments needed for it. Also to make it look better

**Comment by VasilisThePikachu** [comment]
Yeah i never set up a help menu as i never expected someone to actually use this

You are free to modify it as much as you want
